README.md

```diff
@@ -1,18 +1,25 @@
 # Package Analysis
 
-This repo contains a few components to aid in the analysis of open source
-packages, in particular to look for malicious software.
+The Package Analysis project contains components that analyze the capabilities of packages available on open source repositories. The components partiularly look for behaviors that indicate malicious software: 
```

Comment:
Hmm. I think referring to the project as a set of "components" is archaic.

Its probably worth treating the project as a whole. e.g. "The Package Analysis project analyses the capabilities of packages ... The project particularly looks for ..."

Reply:
Updated, sorry for the delay!